backend/GeometryParser.cpp

```diff
@@ -74,32 +100,32 @@ std::unordered_map <std::string, Object*> GeometryParser::parseObject (std::stri
         }
 
         // create object and add to map
-        if (objName == "point" || objName == "Point") {
+        if (objName == "point") {
```

Comment:
Why not also transform objName to lowercase before checking?

Reply:
That's what I am doing in Line 67
`objName = this->nameTranslations[objName];`

Reply:
Oh sorry.. I mixed the changes up for some reason and though you added the capitalized version in this change :roll_eyes:

Reply:
objName comes from the translation map (see line 67)
`objName = this->nameTranslations[objName];`
and the objName here comes from the objectNames vector which is compared to the input string in lowercase (line 58)